### ISSUE TITLE
some kind of error related to handling of \boldsymbol within mediawiki raw text conversion

### ISSUE THREAD

**Issue by federerfanatic** [jgm/texmath#94]
See jgm/pandoc#3362 where it is noted by @jgm that I should file a separate bug against the texmath package. See the zip file in the preceding pandoc bug report.

**Comment by federerfanatic** [comment]
This Haskell stuff is pretty intriguing. What's the advantage of using this over python? Speed? Nice work on the overall project.

**Comment by jgm** [comment]
+++ federerfanatic [Jan 19 17 10:08 ]:
>   This Haskell stuff is pretty intriguing. What's the advantage of using
>   this over python? Speed? Nice work on the overall project.

For me it's really the maintenance advantages of type safety and purity.

The strong type system prevents many kinds of bugs that I'd
have to worry about in Python.  Refactoring is a breeze; I
can change some types and the compiler basically tells me
what I have to change. When I've changed it, I can have a lot
of confidence that I've made all the changes that were
needed.
